CODE_OF_CONDUCT.md

```diff
@@ -5,9 +5,9 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
- contributors and maintainers pledge to making participation in our project
+ contributors and maintainers pledge to make participation in our project
  and our community a harassment-free experience for everyone, regardless of age,
- body size, disability, ethnicity, gender identity and expression,
+ body size, disability, ethnicity, gender identity ,and expression,
```

Comment:
If we keep the comma, it should be on the left of the space.  Do you have a thought why the comma is helpful here? I see it as grouping "gender identity and expression" as a single concept, which seems to make sense to me, but I'm interested in your thoughts.

Reply:
yes, the should have been on the left of the white space.